packages/react-reconciler/src/__tests__/ReactScope-test.internal.js

```diff
@@ -393,6 +393,7 @@ describe('ReactScope', () => {
     });
 
     // @gate www
+    // @gate !disableLegacyMode
```

Comment:
these tests fail when legacy mode is disabled. I am guessing these tests would actually fail if the root was a concurrent root at all and when legacy mode is disabled the test renderer always runs in concurrent mode. Since this is legacy feature that won't be stabilized I decided to turn the tests off when legacy mode is removed